### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Activities_ActivitiesTab_ScreenReader: Voiceover does not read the status message after click on Clear Successful button that what data is cleared.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#2070]
"[ Check out Accessibility Insights! ](https://accessibilityinsights.io/)Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yWCAG2.1; #A11YTCS; #DesktopApp; #MAS4.1.3;#VoiceOver; #Mac;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Navigate to "Activities" Tab present on right bottom of the screen and hit enter.
6. Start Voiceover and using Tab key and check whether the Voiceover reads the status message after click on Clear Successful button,the data has cleared.
**Actual:​**
Voiceover does not read the status message after click on Clear Successful button that what is cleared.

**Expected:**
Voiceover should read the status message after click on clear Successful button,the data has cleared.

**User Impact:**
Screen reader users unable to know that which data has cleared.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
MAS4.1.3:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=0c4efe07-7aa2-430f-849a-57ceed3fee94

**Attachment for Reference:**
[2070_A11y_AzureToolsStorageExplorer_Activities_ActivitiesTab_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3720698/2070_A11y_AzureToolsStorageExplorer_Activities_ActivitiesTab_ScreenReader.pptx)

**Comment by jinglouMSFT** [comment]
@nirmalsahoo  Is this really about Voiceover or NVDA and on which platform?

**Comment by nirmalsahoo** [comment]
@jinglouMSFT this issue is related to Voiceover, Updated the bug description.

**Comment by craxal** [comment]
This is pretty much a duplicate of #1970 because there is no alert of this kind implemented, so it won't work for _any_ screen reader.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Verified the issue on version 1.12.0 and MAC 10.15.3.The issue is still reproduced.

**Comment by nirmalsahoo** [comment]
Verified the issue in version1.12.0 and is no longer reproduced.Hence closing this issue.Please refer the attachment added in onedrive link[]( https://microsoft-my.sharepoint.com/:f:/p/v-nisaho/EsSUKgfoOK9HmEL8PqqXd94BUtAVBzKJetpyH6pKAsUxyw?e=W3kjOv)

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
